dev-client/src/screens/SoilScreen/components/EditSiteSoilDepthPreset.tsx

```diff
@@ -46,44 +44,32 @@ export const EditSiteSoilDepthPreset = ({selected, updateChoice}: Props) => {
   return (
     <Column space="1px" pb="23px">
       <Text variant="body1">{t('soil.soil_preset.info')}</Text>
-      <RadioBlock
-        label={t('soil.soil_preset.label')}
-        labelProps={{variant: 'body1'}}
-        options={{
-          NRCS: {text: t('soil.soil_preset.NRCS')},
-          BLM: {text: t('soil.soil_preset.BLM')},
-          CUSTOM: {text: t('soil.soil_preset.CUSTOM')},
-        }}
-        groupProps={{
-          name: 'soil-preset',
-          value: selectedPreset,
-          onChange: (value: string) =>
-            updateSelectedPreset(
-              value as SoilIdSoilDataDepthIntervalPresetChoices,
-            ),
-        }}
+      <ConfirmModal
+        trigger={onOpen => (
+          <RadioBlock
+            labelProps={{variant: 'body1'}}
+            options={{
+              NRCS: {text: t('soil.soil_preset.NRCS')},
+              BLM: {text: t('soil.soil_preset.BLM')},
+              CUSTOM: {text: t('soil.soil_preset.CUSTOM')},
+            }}
+            groupProps={{
+              name: 'soil-preset',
+              value: selectedPreset,
+              onChange: (value: string) => {
+                updateSelectedPreset(
+                  value as SoilIdSoilDataDepthIntervalPresetChoices,
+                );
+                onOpen();
+              },
+            }}
+          />
+        )}
+        handleConfirm={onConfirm}
+        title={t('projects.inputs.depths.confirm_preset.title')}
+        body={t('projects.inputs.depths.confirm_preset.body')}
+        actionLabel={t('projects.inputs.depths.confirm_preset.confirm')}
       />
```

Comment:
clarification: i think this makes sense, but i don't see this change discussed in the PR description, linked issues, or figma, could you link to the relevant issue?

Reply:
This was part of https://github.com/techmatters/terraso-mobile-client/pull/2823. I have rebased this branch, which should make the "extra" changes go away.

Reply:
ah i see, all good then!